website/www/site/content/en/contribute/release-guide.md

```diff
@@ -244,7 +247,21 @@ __Attention__: Only PMC has permission to perform this. If you are not a PMC, pl
 **********
 
 
-## 2. Create a release branch in apache/beam repository
+## 3. Investigate performance regressions
+
+Check the Beam load tests for possible performance regressions. Measurements are available on [metrics.beam.apache.org](http://metrics.beam.apache.org).
+
+All Runners which publish data should be checked for the following, in both *batch* and *streaming* mode:
+
+- [ParDo](http://metrics.beam.apache.org/d/MOi-kf3Zk/pardo-load-tests) and [GBK](http://metrics.beam.apache.org/d/UYZ-oJ3Zk/gbk-load-test): Runtime, latency, checkpoint duration
+- [Nexmark](http://metrics.beam.apache.org/d/ahuaA_zGz/nexmark): Query runtime for all queries
+- [IO](http://metrics.beam.apache.org/d/bnlHKP3Wz/java-io-it-tests-dataflow): Runtime
+
+If regressions are found, the release branch can still be created, but the regressions should be investigated and fixed as part of the release process.
+JIRA issues should be created for each regression with the 'Fix Version' set to the to-be-released version.
```

Comment:
Who could be the owner of doing a first investigation? Release manager could file issues but may not be in a position to understand the cause of the regressions.

One suggestion for the list of all benchmarks above we add a first point of contact? Would that be reasonable?

Reply:
I thought the release manager would be responsible for identifying and filing the issues in JIRA. After that, they are part of the open issues for the release. The release manager oversees those, but relies on community members to fix them.

It is reasonable to add a point of contact. Problem with these list of contacts is that they tend to become stale (like some of the OWNERS files we have). Perhaps we could just link to the existing OWNERS files? That way, we wouldn't introduce any additional lists.

Reply:
> I thought the release manager would be responsible for identifying and filing the issues in JIRA. After that, they are part of the open issues for the release. The release manager oversees those, but relies on community members to fix them.

I agree.

> It is reasonable to add a point of contact. Problem with these list of contacts is that they tend to become stale (like some of the OWNERS files we have).

My assumption is that if the benchmark exists in this list, it would mean that some people in the community care about it. If nobody is interested in a specific benchmark, and nobody can triage it, it does not need to be listed as one of the release gating important benchmarks list.

> Perhaps we could just link to the existing OWNERS files? That way, we wouldn't introduce any additional lists.

This sounds reasonable to me. I was not was aware that we had OWNERS files and I actually do not know if they are up-to-date.

Alternatively, if a benchmark regression gets no attention during the release process after the release manager filed a jira, and shared in an email etc. we can remove the specific benchmark from this list. What do you think about that? If we do that, we do not need to list specific owners here.

Reply:
I'd like to try filing regression JIRA issues for all listed benchmarks. Hopefully, that should not be too many. Like you said, the issues will be triaged. If there is nobody taking care of an issue, then ultimately we can think about removing the associated benchmark.